src/beignet/nn/functional/_torsion_angle_loss.py

```diff
@@ -0,0 +1,36 @@
+from typing import Tuple
+
+import torch
+from torch import Tensor
+
+
+def torsion_angle_loss(input, target: Tuple[Tensor, Tensor]) -> Tensor:
```

Comment:
Can we call this something like `alphafold2_torsion_angle_loss` and add a reference to alphafold2 supplement algorithm 27 in the docstring?

Reply:
Of course. I will add a docstring!

As far as renaming, are there multiple torsion angle losses?